packages/yoga/src/Spinner/web/Spinner.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import styled, { withTheme } from 'styled-components';
+import {
+  oneOf,
+  string,
+  oneOfType,
+  number,
+} from 'prop-types';
+import get from 'lodash.get';
+
+const StyledSpinner = styled.div`
```

Comment:
This using a svg, which may not be needed.
We can achieve a similar result using only css:

https://github.com/gympass/partners-reports-mfe/blob/82c0fc65c6c4810ff05e14c4cca03b1cbe389592/src/components/atoms/Loading/Loading.style.ts#L60

Check if this can help :)